src/otx/recipe/segmentation/segnext_s.yaml

```diff
@@ -161,3 +161,4 @@ model:
     input_width: 544
     mean: ${..otx_model.config.data_preprocessor.mean}
     std: ${..otx_model.config.data_preprocessor.std}
+    via_onnx: true
```

Comment:
This is a temporal solution, I believe that detail can be wrapped in model-specific export later, rather than exposing to config